packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebViewClient.java

```diff
@@ -199,6 +206,13 @@ public void onUnhandledKeyEvent(WebView view, KeyEvent event) {
         // handled even though they were handled. We don't want to propagate those as they're not
         // truly lost.
       }
+
+      @Override
+      public void onReceivedSslError(WebView view, SslErrorHandler handler, SslError er) {
+        if (FlutterWebViewClient.this.ignoreSslCertificateErrors) {
+          handler.proceed();
+        }
```

Comment:
As a quick note: you're never calling anything on `handler` in the normal case, which I believe will cause requests to hang. I'm not very familiar with this API, but I'm pretty sure you are supposed to be calling `cancel` if you aren't proceeding.